shell/client/accounts/account-settings.html

```diff
@@ -69,6 +69,11 @@ <h3 class="title-bar">Linked identities</h3>
         Log out all other sessions
       </button>
     </p>
+    <p>
+      <button class="account-button delete-account" disabled={{logoutOtherSessionsInFlight}}>
```

Comment:
We should hide this button for users inside the SfW organization -- the can't self-delete, only the admin can delete them.